### PR TITLE
Add test for data copier with none and invalid features

### PR DESCRIPTION
**Pull request by RaghavaAlajangi**
This PR aims to add test functions for `dclab.rtdc_dataset.copier` to improve the code coverage.

### CODE REVIEW

File: tests/test_rtdc_copier.py [hunk 1]
```diff
@@ -230,3 +230,36 @@ def test_copy_specified_feature_list():
...
+    # Make sure this worked
+    with h5py.File(path_copy) as hc:
+        assert "events" not in hc
+
+
+def test_copier_with_wrong_feature():
+    path = retrieve_data("fmt-hdf5_image-bg_2020.zip")
+    path_copy = path.with_name("test_copy.rtdc")
+
+    with pytest.raises(ValueError):
```
Comment (paulmueller) [inline tests/test_rtdc_copier.py:257]: Please add e.g. `match="must be either a list of feature names"` to make sure the correct `ValueError` is raised.

File: tests/test_rtdc_copier.py [hunk 2]
```diff
@@ -230,3 +230,36 @@ def test_copy_specified_feature_list():
...
+        # copy
+        with h5py.File(path) as h5, h5py.File(path_copy, "w") as hc:
```
Comment (paulmueller) [inline tests/test_rtdc_copier.py:259]: - Please switch the pytest and h5py contexts (pytest.raises should be as close to the statement in question as possible)
- After the pytest context ends, please additionally test whether the "events" group is not in the output file

**Review by paulmueller — COMMENTED** [review]
Thanks for the tests. Just a few minior changes.

**Review by paulmueller — APPROVED** [review]
